lib/common/multithread.c

```diff
@@ -29,6 +29,13 @@ struct st_h2o_multithread_queue_t {
 #if H2O_USE_LIBUV
     uv_async_t async;
 #else
+#if defined(__linux__) && LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 27)
```

Comment:
Let's assume that on Linux `eventfd` is always available as a wrapper of the eventfd2 system call. It is supported since linux 2.6.27 + glibc 2.9, as well as from the ancient versions of bionic.

In other words, it would be fair to argue that it would be available on practically all platforms that provide `pipe2`. Therefore switching from using `pipe2` to `eventfd` on linux unconditionally would not cause issues on our users.

Reply:
Agree, i thought should consider old versions below  linux 2.6.27 + glibc 2.9.